client/src/styles/no-match.css

```diff
@@ -9,3 +9,13 @@
 .m-button {
   font-size: 100%;
 }
+
+.m-button:hover{
```

Comment:
Why don't you use standard button from theme?

Why you add this code in this PR? If you want to change this button style, you should create task and make independent PR

Reply:
Yes. Next time. 
Thanks